src/inc/quic_datapath.h

```diff
@@ -141,6 +141,15 @@ typedef struct CXPLAT_SEND_DATA CXPLAT_SEND_DATA;
 //
 typedef struct QUIC_BUFFER QUIC_BUFFER;
 
+//
+// When state is Resolved, LocalLinkLayerAddress and NextHopLinkLayerAddress of CXPLAT_ROUTE are valid.
+//
+typedef enum CXPLAT_ROUTE_STATE {
```

Comment:
I believe we plan to have something like a `RouteSuspect` state very soon (cc @maolson-mstf). I think it'd be a good idea to define it now.

Reply:
I'd rather see that state added in the later PR that actually uses it.

Reply:
Ok.